core/storage/user/gae_models.py

```diff
@@ -131,6 +131,10 @@ class UserSubscriptionsModel(base_models.BaseModel):
     collection_ids = ndb.StringProperty(repeated=True, indexed=True)
     # IDs of feedback thread ids that this user subscribes to.
     feedback_thread_ids = ndb.StringProperty(repeated=True, indexed=True)
+    # IDs of the learners who have subscribed to this user.
+    subscriber_ids = ndb.StringProperty(repeated=True, indexed=True)
+    # IDs of the creators to which this user has subscribed.
+    subscribed_to_ids = ndb.StringProperty(repeated=True, indexed=True)
```

Comment:
creator_ids might be better, to follow the existing trend of collection_ids and feedback_thread_ids.